### ISSUE TITLE
Make options.limit configurable

### ISSUE THREAD

**Issue by Hurtak** [twopluszero/next-images#3]
Feature request: make `options.limit` configurable

**Comment by arefaslani** [comment]
This feature is added in version 0.10.0

**Comment by Hurtak** [comment]
Thanks.

Unfortunately it seems the commit broke path were image lives 
https://github.com/arefaslani/next-images/commit/cca3ee0fa65cb4f6e408a157e3fd158d8423b11d

```diff
-              publicPath: `${assetPrefix}/_next/static/images/`,
+              publicPath: `${assetPrefix}/_next/`,
```

When I take a look inside my `.next` I see the image lives in `.next/static/images`.

**Comment by arefaslani** [comment]
Oh sorry! I will correct this now.

**Comment by arefaslani** [comment]
corrected in 0.10.1 patch.
